apisix/core/table.lua

```diff
@@ -59,6 +59,22 @@ function _M.set(tab, ...)
 end
 
 
+function _M.try_read_attr(tab, ...)
```

Comment:
Should also merge the `try_attr` in other plugin to this function.

Reply:
Sorry, I forgot.